pkg/manifests/manifests.go

```diff
@@ -222,6 +235,13 @@ func (f *Factory) RouterServiceInternal(cr *ingressv1alpha1.ClusterIngress) (*co
 	}
 	s.Spec.Selector["router"] = name
 
+	if f.installConfig.Platform.AWS != nil {
+		if s.Annotations == nil {
```

Comment:
This is not needed, `s.Annotations` is already initialized here: https://github.com/openshift/cluster-ingress-operator/blob/1924e280f10d7457468de81112fbaf8952e1faac/pkg/manifests/manifests.go#L228

Reply:
D'oh yeah!! Am still zonked with all the meds. Fixing now ... Thanks.